crates/api/src/builder/simulator/optimistic_simulator.rs

```diff
@@ -156,37 +147,37 @@ impl<A: Auctioneer, DB: DatabaseService> BlockSimulator for OptimisticSimulator<
         builder_info: &BuilderInfo,
         is_top_bid: bool,
         sim_result_saver_sender: Sender<DbInfo>,
-        request_id: Uuid,
     ) -> Result<bool, BlockSimError> {
         if self.should_process_optimistically(&request, builder_info).await {
             SimulatorMetrics::sim_count(true);
 
             debug!(
-                request_id=%request_id,
```

Comment:
Why is this being removed? request ids are one of the most useful was to connect related logs messages when debugging an issue.